packages/components/src/components/es-loading-text/es-loading-text.css

```diff
@@ -6,4 +6,18 @@
     white-space: pre;
     color: transparent;
     user-select: none;
+    max-width: 100%;
```

Comment:
Does this solve the square end clipping issue?

Reply:
![2024-06-24 at 13 29 36 - Emerald Cicada](https://github.com/EventStore/Design-System/assets/25894545/d95e2040-f045-4ed8-925b-b4575e71dec2)

Reply:
![2024-06-24 at 13 45 16 - Amaranth Tahr](https://github.com/EventStore/Design-System/assets/25894545/f4a45055-e9f5-4ee5-a971-05afac018015)